smriprep/interfaces/freesurfer.py

```diff
@@ -99,7 +99,7 @@ class _ReconAllInputSpec(fs.preprocess.ReconAllInputSpec):
             "parcstats2",
             "cortparc3",
             "parcstats3",
-            "label-exvivo-ec",
+            "label-exvivo-ec-avg",
```

Comment:
We have some 5.3 and 6.0 tags. If this ever gets merged back into nipype, it would be good to not throw away 5.3 support needlessly.

```suggestion
            "label-exvivo-ec",  # 5.3
            "label-exvivo-ec-avg",  # 6.0+
```

Reply:
will this ignore using the `-<no>label-exvivo-ec` flag on FS6+? I did not realize the commented sections were still being used...

Reply:
This just lists the valid steps. I'm not sure where `-<no>label-exvivo-ec` is being used. Could you point to the problem that's happening somewhere?

Reply:
So the problem is that `parcstats` is producing the command line:
```
-autorecon-hemi lh -nohyporelabel -lh-only -openmp 8 -subjid sub-XXXXXX -sd /out/sourcedata/freesurfer \
-notessellate -nosmooth1 -noinflate1 -noqsphere -nofix -nowhite -nosmooth2 -noinflate2 -nocurvstats \
-nosphere -nosurfreg -nojacobian_white -noavgcurv -nocortparc -nopial -nopctsurfcon -noparcstats \
-nocortparc2 -noparcstats2 -noparcstats3 -nolabel-exvivo-ec
```

and the `-<no>label-exvivo-ec` flag is no longer valid, causing an instant crash.

Reply:
Are you able to reproduce this in a debug session? I can't see why it would be happening, since `label-exvivo-ec` is only used [here](https://github.com/nipy/nipype/blob/4a91e88777465b435935d4dcaceff8a8345ba2dd/nipype/interfaces/freesurfer/preprocess.py#L1201), which is guarded by `if Info.looseversion() < LooseVersion("6.0.0"):`.

Reply:
Ah, I am able to reproduce this for nibabies, and see the problem - essentially `Info.looseversion() == 0.0.0` (since the developmental build-stamp.txt is not being correctly parsed), which causes this flag to be included when forcing `recon-all`.

I do not know why this is happening with fmriprep (https://github.com/nipreps/fmriprep/issues/2832) - we'll need to collect / run the `command.txt` and debug that.

Reply:
I believe so, but I have been using a prebuilt distribution instead of building FS (+ the infant module) from source.